### DISCUSSION TITLE
repo already added Lists-Name as tags and more..

### DISCUSSION THREAD

**Discussion by Entwineo** [community/community#34854] [Other Feature Feedback, Questions, & Ideas]
Suggestions.
1. Make easy to find how many list(Private) have been assigned to particular repo.
2. Custom Reorder & color list manually, auto.
3. Filter by tag, keyboard, updated etc.
4. Make List size small it is taking too much space on screen vertically

**Comment by vrad892** [comment]
> Suggestions.
> 1. Make easy to find how many list(Private) have been assigned to particular repo.
> 2. Custom Reorder & color list manually, auto.
> 3. Filter by tag, keyboard, updated etc.
> 4. Make List size small it is taking too much space on screen vertically

**Comment by Javidnewlife** [comment]
> Suggestions.
> 1. Make easy to find how many list(Private) have been assigned to particular repo.
> 2. Custom Reorder & color list manually, auto.
> 3. Filter by tag, keyboard, updated etc.
> 4. Make List size small it is taking too much space on screen vertically

**Comment by Javidnewlife** [comment]
> > Suggestions.
> > 1. Make easy to find how many list(Private) have been assigned to particular repo.
> > 2. Custom Reorder & color list manually, auto.
> > 3. Filter by tag, keyboard, updated etc.
> > 4. Make List size small it is taking too much space on screen vertically
> 
>
